casepro/msgs/forms.py

```diff
@@ -84,8 +84,8 @@ class Meta:
 
 class FaqForm(forms.ModelForm):
 
-    question = forms.CharField(label=_("Question"), max_length=140)
-    answer = forms.CharField(label=_("Answer"), max_length=140)
+    question = forms.CharField(label=_("Question"), max_length=255)
```

Comment:
It could be, the questions they load seem to be quite short but I think that you're right and it will make more sense as a TextArea.